src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/naming/abbreviationaswordinname/InputAbbreviationAsWordInNameCheckEnhancedInstanceof.java

```diff
@@ -3,25 +3,29 @@
 
 import java.util.ArrayList;
 import java.util.Locale;
+// config:
+// allowedAbbreviationLength="1"
 
 public class InputAbbreviationAsWordInNameCheckEnhancedInstanceof {
```

Comment:
multiline comment please as in other files.

Reply:
Fixed.